templates/core/services/thank-you.html

```diff
@@ -2,49 +2,50 @@
 
 {% block title %}Thank you | AWS{% endblock %}
 
-{% block head_extra%}<meta name="robots" content="noindex" />{% endblock %}
+{% block head_extra %}<meta name="robots" content="noindex" />{% endblock %}
 
 {% block content %}
-<div class="p-strip is-bordered is-deep">
-  <div class="row">
-    <div class="col-7">
-      <h1>Thank you for your interest in SMART START</h1>
-      <p class="p-heading--4">A member of our team will be in touch within one working day.</p>
-    </div>
-    <div class="col-5 u-vertically-center u-hide--medium u-hide--small u-align--center">
-      {{
-        image(
-          url="https://assets.ubuntu.com/v1/52d53696-picto-thankyou-midaubergine.svg",
-          alt="",
-          width="200",
-          height="200",
-          hi_def=True,
-          loading="auto"
-        ) | safe
-      }}
+  <div class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
+        <h1 class="u-no-margin--bottom">Thank you for your interest in SMART START.</h1>
+        <h2>A member of our team will be in touch within one working day.</h2>
```

Comment:
@Sophie-32 left [this](https://github.com/canonical/ubuntu.com/pull/14225#issuecomment-2307330137) comment for a couple of the thank-you pages I worked on recently, apparently for verbiage these pages should be changed as follows: 
```suggestion
        <h2>A member of our team will be in touch shortly.</h2>
```